### PR TITLE
Kachurovska Yuliia, compmat 1, lab2: 3.1 - 3.7

### PR DESCRIPTION
**Pull request by yuliakachurovska**

### CODE REVIEW

File: lab3/lab3.c [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+void task7_a() {
+    double a, b, c;
+    double x1, x2, D;
+    printf("ax^2+bx+c. Enter value a, b, c = ");
+    scanf("%lf %lf %lf", &a, &b, &c);
+    D = (b*b) - (4*a*c);
+    if (D < 0) {
+        printf("There are no solutions!");
+    }
+    if (D == 0) {
```
Comment (mecchmatProjects) [inline lab3/lab3.c]: double comaparison

File: lab3/lab3.c [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    }
+    printf("Min number = %.4lf", min_number);
+
+}
+
+void task7_a() {
+    double a, b, c;
+    double x1, x2, D;
+    printf("ax^2+bx+c. Enter value a, b, c = ");
+    scanf("%lf %lf %lf", &a, &b, &c);
```
Comment (mecchmatProjects) [inline lab3/lab3.c:130]: xcase a==0?

Comment (yuliakachurovska) [reply lab3/lab3.c:130]: I added the changes to the file. Now there is a check where a == 0

File: lab3/lab3.c (hunk 1 above)
Comment (yuliakachurovska) [reply lab3/lab3.c]: Can you explain, I don't understand

**Review by mecchmatProjects — APPROVED** [review]

**Pull request merged**